provider/keycloak_custom_user_federation_test.go

```diff
@@ -0,0 +1,117 @@
+package provider
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/mrparkers/terraform-provider-keycloak/keycloak"
+	"regexp"
+	"testing"
+)
+
+func TestAccKeycloakCustomUserFederation_basic(t *testing.T) {
+	skipIfEnvSet(t, "CI") // temporary while I figure out how to load this custom provider in CI
```

Comment:
Thoughts on a custom Keycloak image with the provider pre-installed?

Reply:
It's either that, or setting up an entrypoint script that `wget`s the jar directly from github and places it in the correct folder.  I'm not sure I'm a fan of maintaining both a jar and a custom docker image just for these tests.